### PR TITLE
feature/add-stealth-mode-ability

### PR DESCRIPTION
**Pull request by ferrom-jaia**
Added ability to turn stop sending bot status messages, disable WIFI, disable bluetooth

### CODE REVIEW

File: src/bin/mission_manager/machine.cpp [hunk 1]
```diff
@@ -269,8 +269,10 @@ jaiabot::statechart::inmission::underway::Task::~Task()
 
     task_packet_.set_end_time_with_units(goby::time::SystemClock::now<goby::time::MicroTime>());
 
-    if (task_packet_.type() != protobuf::MissionTask::NONE)
+    if (task_packet_.type() != protobuf::MissionTask::NONE && !this->machine().rf_disable())
+    {
         intervehicle().publish<groups::task_packet>(task_packet_);
+    }
```
Comment (jason-jaia) [inline src/bin/mission_manager/machine.cpp:275]: You probably want to publish the task packet to interprocess so we keep logging the task packets

File: src/bin/fusion/fusion.cpp [hunk 2]
```diff
@@ -602,17 +611,75 @@ void jaiabot::apps::Fusion::loop()
...
+                    interprocess().publish<jaiabot::groups::powerstate_command>(
+                        latest_engineering_status);
+                }
+
+                // If the rf_disable timeout has been reach then start sending bot status
+                if ((last_bot_status_report_time_ +
+                     std::chrono::minutes(rf_disabled_timeout_mins_)) <= now)
+                {
+                    // Set bot status rate to 1 Hz if the rf disable timeout is reached
+                    latest_engineering_status.set_bot_status_rate(
```
Comment (jason-jaia) [inline src/bin/fusion/fusion.cpp:674]: We should use the bot status rate from the config file

File: src/doc/markdown/page80_timing.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ This following table summarizes the time intervals for things like polling, logg
...
 |                                           | Send/Receive UDP to `jaiabot_web_portal` across network                             | Depends on network latency                      |                                                               | 08/25/2022
 | jaiabot_web_portal                        | Send/Receive via intervehicle to `jaiabot_mission_manager` or `jaiabot_engineering` | Depends on XBee radio latency                   | src/bin/                                                      | 08/25/2022
-| jaiabot_driver_arduino                    | Send/Receive ArduinoCommand via serial (each loop steps the motor throttle)         | 0.25 s loop interval                            | src/bin/drivers/arduino/app.cpp                               | 08/25/2022
+| jaiabot_driver_arduino                    | Send/Receive ArduinoCommand via serial (each loop steps the motor throttle)         | 0.1 s loop interval                            | src/bin/drivers/arduino/app.cpp                               | 08/25/2022
```
Comment (jason-jaia) [inline src/doc/markdown/page80_timing.md]: Update date field to indicate when you changed it

File: src/doc/markdown/page80_timing.md [hunk 4]
```diff
@@ -24,7 +24,7 @@ This following table summarizes the time intervals for things like polling, logg
...
 | jaiabot_engineering app                   | Publish Engineering status loop                                                     | 5 s loop interval                               | src/bin/jaiabot_engineering/app.cpp                           | 08/25/2022
 | jaiabot_metadata app                      | No loop interval                                                                    | No loop interval                                | src/bin/jaiabot_metadata/app.cpp                              | 08/25/2022
-| jaiabot_pid_control app                   | Publish low control loop                                                            | 0.5 s loop interval                             | src/bin/jaiabot_pid_control/app.cpp                           | 08/25/2022
+| jaiabot_pid_control app                   | Publish low control loop                                                            | 0.1 s loop interval                             | src/bin/jaiabot_pid_control/app.cpp                           | 08/25/2022
```
Comment (jason-jaia) [inline src/doc/markdown/page80_timing.md]: Update date field to indicate when you changed it

File: src/web/command_control/client/components/PIDGainsPanel.tsx [hunk 5]
```diff
@@ -211,6 +208,20 @@ export class PIDGainsPanel extends React.Component {
...
+                                    <td key="rf_disable_timeout_mins_label">Update RF Disable Time Mins</td>
+                                    <td>
+                                        <input style={{maxWidth: "80px"}} 
+                                            type="number" 
+                                            id="rf_disable_timeout_mins_input" 
+                                            name="rf_disable_timeout_mins_input" 
+                                            defaultValue={engineering?.rf_disable_options?.rf_disable_timeout_mins ?? "-"}
+                                            min="1"
+                                            max="255"
+                                            step="any"
```
Comment (jason-jaia) [inline src/web/command_control/client/components/PIDGainsPanel.tsx]: Why are we using any, prob should be step 1

File: src/web/command_control/client/components/QueryBotStatusPanel.tsx [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+                            <td>Bot Id</td>
+                            <td>
+                                <input style={{maxWidth: "80px"}} 
+                                    type="number" 
+                                    id="query_bot_status_input" 
+                                    name="query_bot_status_input" 
+                                    defaultValue="0"
+                                    min="0"
+                                    max="30"
+                                    step="any"
```
Comment (jason-jaia) [inline src/web/command_control/client/components/QueryBotStatusPanel.tsx]: Step = 1

File: src/bin/fusion/fusion.cpp (hunk 2 above)
Comment (ferrom-jaia) [reply src/bin/fusion/fusion.cpp:674]: Added a todo... I need to add a function that reads in the enum and then sets the bot_status_period

I will create ticket.

**Review by jason-jaia — APPROVED** [review]
Walked through with Matt and saw in simulation.

**Pull request merged**